test/js/asyncprogressworkerstream-test.js

```diff
@@ -9,9 +9,14 @@
 const test     = require('tap').test
     , testRoot = require('path').resolve(__dirname, '..')
     , bindings = require('bindings')({ module_root: testRoot, bindings: 'asyncprogressworkerstream' })
-    , Readable = require('stream').Readable
     , util = require('util');
 
+var Readable
+if (process.versions.node.split('.')[1] <= 8)
```

Comment:
Add a conjunction on the major being 0, else this would trigger for any minor version of 8, e.g. a possible 4.8.